Zend/zend_compile.h

```diff
@@ -381,6 +395,7 @@ typedef struct _zend_property_info {
 	HashTable *attributes;
 	zend_class_entry *ce;
 	zend_type type;
+	zend_function **accessors;
```

Comment:
Is this some sort of memory optimization or why double indirect pointer instead of `zend_function *accessors[ZEND_ACCESSOR_COUNT];`?

Reply:
I was thinking ahead to when we have more accessors, and didn't want to allocate storage for them unconditionally. A bit weird if only two are involved.

Maybe a nice option would be to use a flexible array member.